tests/src/test/java/org/eclipse/hono/tests/registry/TenantHttpIT.java

```diff
@@ -164,6 +165,38 @@ public void testAddTenantSucceedsForEmptyBody(final VertxTestContext context) {
         .setHandler(context.completing());
     }
 
+    /**
+     * Verifies that the service successfully create a tenant from a request without a body.
+     *
+     * @param context The vert.x test context.
+     */
+    @Test
+    public void testAddTenantSucceedsWithNoRequestBody(final VertxTestContext context) {
+
+        registry.addTenant(tenantId)
+                .setHandler(context.completing());
+    }
+
+    /**
+     * Verifies that the service successfully create a tenant with a generated tenant ID.
+     *
+     * @param context The vert.x test context.
+     */
+    @Test
+    public void testAddTenantSucceedsWithAGeneratedId(final VertxTestContext context) {
```

Comment:
since you also allow to create a device without providing a body, shouldn't we also add a corresponding test case to the suite for the `/devices` resources?

Reply:
I added a test